internal/services/contracts_events_consumer.go

```diff
@@ -140,10 +140,10 @@ func (c *ContractsEventsConsumer) routeTransferEvent(e *ContractEventData) error
 	case common.HexToAddress(c.settings.AftermarketDeviceContractAddress):
 		return c.handleAfterMarketTransferEvent(e)
 	default:
-		c.log.Debug().Str("event", e.EventName).Interface("fullEventData", e).Msg("Handler not provided for event.")
+		c.log.Debug().Str("event", e.EventName).Interface("fullEventData", e).Msg("Handler not provided for contract")
 	}
 
-	return nil
+	return errors.New("Handler not provided for contract")
```

Comment:
I'm not sure that this qualifies as an _error_. It's expected that `Transfer` events come in that we don't handle.

The enclosing log statement is at the debug level, so I guess it doesn't matter. Do we need _both_ this and the log line above, though? I figured you did this to cut down on the log statements.